server/executor/trigger/traceid.go

```diff
@@ -30,6 +32,10 @@ func (t *traceidTriggerer) Type() trigger.TriggerType {
 }
 
 func (t *traceidTriggerer) Resolve(ctx context.Context, test test.Test, opts *ResolveOptions) (test.Test, error) {
+	if test.Trigger.Type == trigger.TriggerTypeCypress {
```

Comment:
Can we match this against a list? this can also happen for future e2e frameworks